test-data/unit/check-final.test

```diff
@@ -1082,3 +1082,10 @@ class A:
         self.x = 10  # type: Final
         undefined  # type: ignore
 [builtins fixtures/tuple.pyi]
+
+[case testFinalUsedWithClassVar]
+from typing import Final, ClassVar
+
+class A:
+    a: Final[ClassVar[int]]  # E: Variable should not be annotated with both ClassVar and Final
```

Comment:
Can you test some other variants, such as `a: ClassVar[Final] = 1`, `a: ClassVar[Final[int]]`?

Reply:
Such cases will produce an error with message:
```
Final can be only used as an outermost qualifier in a variable annotation.
```

Should it be changed to `Variable should not be annotated with both ClassVar and Final`?

Reply:
If it's easy to do, then yes. The existing error message would make users first swap the qualifiers, then get an error anyway from your new check, which is not very user-friendly. But don't worry about changing the error message if it's complicated.

Either way I think it would be useful to test all these variations here. They can all go in the same test case.

Reply:
Looks like it is not easy to change message for a `ClassVar[Final[int]]` case.

I will skip this for this moment.